mixins/localize-mixin.md

```diff
@@ -4,9 +4,9 @@ The `LocalizeMixin` and `LocalizeStaticMixin` allow you to localize text in your
 
 ## Providing Resources
 
-Your component must provide resources by either implementing a static `resources` getter for local resources, or a `getLocalizeResources` method to fetch resources asynchronously. The `getLocalizeResources` method will be passed an array of lowercase languages in preferential order.
+Your component must provide resources by either implementing a static `resources` getter for local resources, or a `config` getter to fetch resources asynchronously. The `importFunc` method of `config` will be called with lowercase languages in preferential order.
 
-Your implementation should find the resources that best match the languages passed in. It should then return an object containing two values:
+Static implementations should return an object containing two values:
```

Comment:
Since it only applies to static, can you move this bit down to the static example?

Reply:
This appears to not even be accurate for static implementations?

Reply:
Oh weird yeah they don't need to return anything eh.